AppDashboard/templates/users/login.html

```diff
@@ -34,7 +34,11 @@
                   <div style="clear: both"></div>
                   <div style="margin: 15px 0 20px 9px">
                     {% if show_create_account %}
-                    <div>Don't have an account? <a href="/users/new">Create one.</a></div>
+                        {% if continue %}
+                            <div>Don't have an account? <a href="/users/new?continue={{ continue }}">Create one.</a></div>
```

Comment:
Since this allows user input, it should be sanitized. For example, a user could submit `continue_url"><script>something malicious</script><a href="#` as a continue url.